### ISSUE TITLE
[détail] Le bouton "commencer" de la home devient useless une fois l'utilisateur connecté

### ISSUE THREAD

**Issue by damienlethiec** [Willgg/techcare#149]
Ce serait cool de le rediriger vers son dashboard ou une autre vue utile
![image](https://cloud.githubusercontent.com/assets/8105430/18646997/4fc90ada-7eb4-11e6-8d38-e610579d14df.png)

**Comment by Willgg** [comment]
Tout à fait. Le user devrait être rédiger vers son dashboard quand il tente "/" root_path
